Sources/FoundationExtensions/Bootstrap/DistributionEnvironment.swift

```diff
@@ -27,7 +27,7 @@ public enum DistributionEnvironment: String {
 
     public static var current: DistributionEnvironment {
         // We are passing the launch argument `-TESTING` when running
-        // the TeufelBluetoothTests.
+        // the Tests.
         #if DEBUG
         if ProcessInfo.processInfo.arguments.contains("-TESTING") {
```

Comment:
shall we rename TeufelBuildType to something more generic?

Reply:
I removed this DistributionEnvironment completely from FoundationExtensions. I guess this is more an internal thing and shouldn't be here. I'll move it to some internal lib.